Tools/scripts/uploader.py

```diff
@@ -316,8 +317,12 @@ def __sync(self):
         # that we might still have in progress
         # self.__send(uploader.NOP * (uploader.PROG_MULTI_MAX + 2))
         self.port.flushInput()
+        # on WSL disconnected bluetooth ports will block trying to write
+        self.port.write_timeout = 0.1
```

Comment:
Maybe create a `old_write_timeout = self.port.write_timeout` and use it instead of `None`.  Will create future breakage.

Reply:
Fixed